ports/rp2/boards/W55RP20_EVB_PICO/readme.md

```diff
@@ -0,0 +1,22 @@
+# Wiznet W55RP20-EVB-Pico
+
+## Network Example
+
+To use network / socket based code, connect ethernet port to network with DHCP running:
+
+```
+>>> import network
+>>> from machine import Pin, WIZNET_PIO_SPI
+>>> spi = WIZNET_PIO_SPI(baudrate=10_000_000, sck=Pin(18), cs=Pin(17), mosi=Pin(19), miso=Pin(16))
```

Comment:
Are you sure about these pins for the W55RP20? These are the pins for the W5500 (PICO/PICO2). I think for the W55RP20 the pins should be:

```suggestion
>>> spi = WIZNET_PIO_SPI(baudrate=10_000_000, sck=Pin(21), cs=Pin(20), mosi=Pin(23), miso=Pin(22))
```